client/web/web_test.go

```diff
@@ -77,17 +77,13 @@ func TestQnapAuthnURL(t *testing.T) {
 func TestServeAPI(t *testing.T) {
 	lal := memnet.Listen("local-tailscaled.sock:80")
 	defer lal.Close()
-
 	// Serve dummy localapi. Just returns "success".
-	go func() {
-		localapi := &http.Server{Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			fmt.Fprintf(w, "success")
-		})}
-		defer localapi.Close()
-		if err := localapi.Serve(lal); err != nil {
-			t.Error(err)
```

Comment:
data race + failure was due to this returning an error in a separate goroutine after the test had returned